habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/ContainerizeDepsMojo.java

```diff
@@ -264,4 +288,31 @@ protected void setAnchorSourceDirectory(File anchorSourceDirectory) {
     public File getAnchorOutputDirectory() {
         return anchorOutputDirectory;
     }
+
+    protected void setDockerfile(File dockerfile) {
+        this.dockerfile = dockerfile;
+    }
+
+    protected void performDockerfileUpdateForVirtualEnvironment() {
+        String updatedDockerfile =
+                DockerfileUtil.updateDockerfileWithContainerStageLogic(dockerfile,
```

Comment:
For clarity - is the dockerfile variable on this line the class property?  i.e. `this.dockerfile`?  If so can you tack on `this.` 
It's not a big deal, but would make it a little more readable.